docs/Operators.md

```diff
@@ -3283,20 +3283,20 @@ This version of the operator has been available since version 10 of the default
 #### Inputs (2 - 4)
 
 <dl>
-<dt><tt>x</tt> : T1</dt>
+<dt><tt>x</tt> (non-differentiable) : T1</dt>
```

Comment:
Why is it non-differentiable? To speed up the merge, we probably should split this PR into two PRs (one for normal ops and the other one for quantized ops)?

Reply:
Removed the differentiable variables for QlinearConv and ConvInteger operators